src/components/Loader.vue

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
copyright header ;)

Reply:
Is that really needed? I added it, but one LICENSE at the root of the project looks enough to me.

Reply:
Well, that's how we do things here :)